.github/actions/build-ciab/build-ciab.js

```diff
@@ -32,10 +32,7 @@ function moveRPMs() {
 		.filter(item => fs.lstatSync(item).isDirectory()) // get a list of directories within dist
 		.flatMap(directory => fs.readdirSync(directory).map(item => path.join(directory, item))) // list files within those directories
 		.filter(item => /\.rpm$/.test(item)) // get a list of RPMs
-		.map(rpm => {
-			fs.renameSync(rpm, rpm.replace(new RegExp('.*/'), ''));
-			return rpm;
-		}); // move the RPMs to the dist folder
+		.map(rpm => fs.renameSync(rpm, rpm.replace(new RegExp('.*/'), ''))); // move the RPMs to the dist folder
```

Comment:
Will this run on Windows executors? I mean, I know it _won't_ for the foreseeable future, but if you tried would it work?

Reply:
It would not. Changed to using `path.basename()` in 0e1a70389a